ignite/metrics/accuracy.py

```diff
@@ -161,11 +163,12 @@ def update(self, output: Sequence[torch.Tensor]) -> None:
             y = torch.transpose(y, 1, last_dim - 1).reshape(-1, num_classes)
             correct = torch.all(y == y_pred.type_as(y), dim=-1)
 
-        self._num_correct += torch.sum(correct).item()
+        # Don't need to detach here because torch.eq is not differentiable, so the computation graph is detached anyway.
```

Comment:
It should be more readable to explicitly detach tensor.

Reply:
IMO, it is OK to add the comment only here